backend/src/app/nitrate_client.clj

```diff
@@ -0,0 +1,152 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) KALEIDOS INC
+
+(ns app.nitrate-client
+  "Module that make calls to the external nitrate aplication"
+  (:require
+   [app.common.logging :as l]
+   [app.common.schema :as sm]
+   [app.config :as cf]
+   [app.http.client :as http]
+   [app.rpc :as-alias rpc]
+   [app.setup :as-alias setup]
+   [app.util.cache :as cache]
+   [app.util.json :as json]
+   [app.worker :as-alias wrk]
+   [clojure.core :as c]
+   [integrant.core :as ig]))
+
+
+;; TODO Extract to env
+(def baseuri "http://localhost:3000")
+
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; HELPERS
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(defn- create-cache
+  [{:keys [::wrk/executor]}]
+  (cache/create :executor executor
+                :keepalive "5m"
+                :max-size 1000))
```

Comment:
We should discuss about this, this cache works OK on single instance but it will not work well when we run multiple instances.